### PR TITLE
Adapt chart to use slim images

### PR DESCRIPTION
**Pull request by tompizmor**
Change chart to use the bitnami slim images for tensorflow inception model

### CODE REVIEW

File: chart/tensorflow/templates/NOTES.txt [hunk 1]
```diff
@@ -1,11 +1,8 @@
...
-$ curl -o 'cat-over-keyboard.jpg' 'http://www.rd.com/wp-content/uploads/sites/2/2016/04/01-cat-wants-to-tell-you-laptop.jpg'
-$ bazel-bin/tensorflow_serving/example/inception_client --server=localhost:9000 --image=cat-over-keyboard.jpg
+Get the port from the service list:
+$ kubectl get svc
+NAME                 CLUSTER-IP   EXTERNAL-IP   PORT(S)          AGE
+tensorflow-serving   10.0.0.233   <nodes>       9000:32725/TCP   5m
+
+Test it:
+$ docker run --rm -it -v ~/Downloads/:/tmp/ bitnami/tf-client-slim:0.0.1
+$ bazel-bin/tensorflow_serving/example/inception_client  --server=192.168.99.100:32725 --image=/tmp/labrador-retriever-dog-pictures-6.jpg
```
Comment (sebgoa) [inline chart/tensorflow/templates/NOTES.txt]: you don't  drop as root here ?

File: chart/tensorflow/templates/inception.yaml [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tf
+    spec:
+      initContainers:
+      - name: download-data
+        image: bitnami/minideb-extras:latest
+        command: ["/bin/sh", "-c", "curl -o /seed/inception-v3-2016-03-01.tar.gz http://download.tensorflow.org/models/image/imagenet/inception-v3-2016-03-01.tar.gz && cd /seed/ && tar -xzf inception-v3-2016-03-01.tar.gz && rm inception-v3-2016-03-01.tar.gz"]
```
Comment (sebgoa) [inline chart/tensorflow/templates/inception.yaml]: you don't like my Job that puts the model data in a PVC ?
if you put the download in the init, it will redownload the model everytime you scale.

File: chart/tensorflow/templates/NOTES.txt (hunk 1 above)
Comment (tompizmor) [inline chart/tensorflow/templates/NOTES.txt]: yes, you're `root` inside the container. is there any issue with that?

File: chart/tensorflow/templates/inception.yaml (hunk 2 above)
Comment (tompizmor) [inline chart/tensorflow/templates/inception.yaml]: the problem with the Job is that by the time it finishes, the `load-model` init-container has already tried to load a unexistent model. I don't know if it is possible to wait until the Job finished before start with the init-containers in the deployment.

File: chart/tensorflow/templates/NOTES.txt (hunk 1 above)
Comment (sebgoa) [inline chart/tensorflow/templates/NOTES.txt]: the shell is not # but $

File: chart/tensorflow/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ TensorFlow Serving is an open-source software library for serving machine learni
 ## TL;DR;
 
 ```console
-$ https://github.com/tompizmor/tensorflow
+$ https://github.com/bitnami/tensorflow
```
Comment (sebgoa) [inline chart/tensorflow/README.md]: git clone ?

Comment (tompizmor) [inline chart/tensorflow/README.md]: yes
